modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_15_0/ObjectRelationshipUpgradeProcess.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.internal.upgrade.v3_15_0;
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Rodrigo Paulino
+ */
+public class ObjectRelationshipUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		if (!hasColumn("ObjectRelationship", "parameterObjectFieldId")) {
```

Comment:
We don't need to use the `hasColumn` anymore. The `alterTableAddColumn` method is now doing this check (see [LPS-147889](https://issues.liferay.com/browse/LPS-147889))

Reply:
Nice. Didn't know that! Thanks, Carol!